syntax = "proto3";

import "avframe.proto";

enum VtsMsgType {
    VTS_MSG_INVALID = 0;
    VTS_MSG_AVFRAME = 1;
    VTS_MSG_AVSTOP = 2;
    VTS_MSG_HEARTBEAT = 3;
}

message VtsMsg {
  int32 version = 1;
  VtsMsgType type = 2;
  VtsAvFrame avFrame = 3;
}
